app/models/minute.rb

```diff
@@ -26,7 +26,12 @@ def export_to_github_wiki
     git.config('user.email', ENV.fetch('GITHUB_USER_EMAIL'))
 
     filename = "#{title}.md"
-    File.write(File.join(working_directory, filename), to_markdown)
+    filepath = File.join(working_directory, filename)
+    # Minute#titleにユーザーの入力値が含まれるようになった場合、ディレクトリトラバーサルが発生する危険性がある
+    # 現時点でのMinute#titleの実装では問題が発生しないが、念の為ファイル名の確認を行っておく
+    raise 'Error!, Invalid file name.' unless File.dirname(filepath) == working_directory.to_s
```

Comment:
```suggestion
    raise "Error!, Invalid file name: #{filename}" unless File.dirname(filepath) == working_directory.to_s
```

例外メッセージに原因となったファイル名を含めた方が、例外発生時にデバッグがしやすくなります。